pkg/pipelines/scm/github.go

```diff
@@ -53,18 +54,24 @@ func (r *githubSpec) pushBindingName() string {
 
 func (r *githubSpec) prBindingParams() []triggersv1.Param {
 	return []triggersv1.Param{
-		createBindingParam("gitref", "$(body.pull_request.head.ref)"),
-		createBindingParam("gitsha", "$(body.pull_request.head.sha)"),
+		createBindingParam(triggers.GitRef, "$(body.pull_request.head.ref)"),
+		createBindingParam(triggers.GitCommitID, "$(body.pull_request.head.sha)"),
 		createBindingParam("gitrepositoryurl", "$(body.repository.clone_url)"),
```

Comment:
Also, how about grouping all the `triggers.X` params together?

It's coming out like this:

```yaml
  - name: io.openshift.build.commit.id
    value: $(body.pull_request.head.sha)
  - name: gitrepositoryurl
    value: $(body.repository.clone_url)
  - name: fullname
    value: $(body.repository.full_name)
```

Basically, this makes it easier for a human to read, breaking the list of similar items up with two incongruous items makes it harder to understand.